### ISSUE TITLE
polymer init errors out

### ISSUE THREAD

**Issue by atomikolex** [Polymer/polymer-cli#275]
### Description
Trying to polymer init the app-drawer-template or any other init option gives these errors:
```
info:    Running template app-drawer-template
error:   Promise rejection: AssertionError: A name parameter is required to create a storage
error:   AssertionError: A name parameter is required to create a storage
    at new Storage (/usr/local/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/util/storage.js:24:3)
    at GithubGenerator.Base._getStorage (/usr/local/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/base.js:665:10)
    at GithubGenerator.Base (/usr/local/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/base.js:157:22)
    at GithubGenerator (/usr/local/lib/node_modules/polymer-cli/lib/init/github.js:21:36)
    at Environment.instantiate (/usr/local/lib/node_modules/polymer-cli/node_modules/yeoman-environment/lib/environment.js:328:10)
    at Environment.create (/usr/local/lib/node_modules/polymer-cli/node_modules/yeoman-environment/lib/environment.js:305:15)
    at Environment.run (/usr/local/lib/node_modules/polymer-cli/node_modules/yeoman-environment/lib/environment.js:364:24)
    at runGenerator (/usr/local/lib/node_modules/polymer-cli/lib/commands/init.js:65:29)
    at /usr/local/lib/node_modules/polymer-cli/lib/commands/init.js:131:25
```

### Versions & Environment
- Polymer CLI: 0.11.1
- node: 5.1.0
- Operating System: Ubuntu 16.04, 32-bit

#### Steps to Reproduce
polymer init from an empty project folder, select any option

#### Expected Results
No error is thrown 

#### Actual Results
Error is thrown

**Comment by LoveAndHappiness** [comment]
I believe this error has been fixed here: https://github.com/Polymer/polymer-cli/issues/247

And the fix will be released with the next minor version 0.12.0

Is there already a release schedule @justinfagnani for 0.12.0?

**Comment by atomikolex** [comment]
I am getting the same errors, so maybe it's the same issue. I am not running init in an existing project with a present package.json though. Was just following the recent Polycast on the CLI and ran it inside an empty folder.

**Comment by FredKSchott** [comment]
@atomikolex are you able to reproduce by installing from this git repo? (See: http://stackoverflow.com/questions/17509669/how-to-install-nodejs-package-from-github-directly)

@LoveAndHappiness we have one more build feature we'd like to get in before releasing 0.12.0.

**Comment by atomikolex** [comment]
When installing directly from the git repo, polymer init immediately gives the following errors, that is before I get to the template option screen:
module.js:340
    throw err;
    ^

Error: Cannot find module '..'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /usr/local/lib/node_modules/polymer-cli/bin/polymer.js:26:36
    at /usr/local/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:46:14
    at process (/usr/local/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:92:31)

**Comment by atomikolex** [comment]
I can confirm CLI version  0.12.0 solves my init issues for the app-drawer and shop templates. Yay!

Trying to run a blank template application or element still throws the assertion error:  Promise rejection: AssertionError: A name parameter is required to create a storage

**Comment by tjmonsi** [comment]
Hi everyone, I just started creating another polymer project right now and it seems even when using 0.12.0, I still get the same errors when doing polymer init

```
error:   Uncaught exception: Error: Cannot find module 'yeoman-environment'
error:   Error: Cannot find module 'yeoman-environment'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/ubuntu/.nvm/versions/node/v4.4.5/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/index.js:6:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

**Comment by atomikolex** [comment]
@tjmonsi Just a noob, but I think you need to file a separate issue for this. My issue was an assertion error, in your case a specific module isn't there. Maybe double check on installation errors also.

**Comment by tjmonsi** [comment]
@atomikolex it's alright... I thought it was the same error. But then it just got magically fixed (still using the same version). It's just weird though, but when I started to run up another instance of my dev place (I use c9), and just installed polymer-cli, it works like a charm. Just before this I installed polymer-cli bower and firebase-tools, and it went haywire. I can't really replicate it, but well it works now. Sorry about this :)
